src/LinAlg/hiopVectorRajaPar.cpp

```diff
@@ -1165,6 +1205,24 @@ double hiopVectorRajaPar::logBarrier_local(const hiopVector& select) const
   return sum.get();
 }
 
+/**
+ * @brief Sum all elements
+ */
+double hiopVectorRajaPar::sum_local() const
+{
+  double* data = data_dev_;
+  RAJA::ReduceSum< hiop_raja_reduce, double > sum(0.0);
+  RAJA::forall< hiop_raja_exec >(
+    RAJA::RangeSegment(0, n_local_),
+		RAJA_LAMBDA(RAJA::Index_type i)
+    {
```

Comment:
```suggestion
    RAJA::RangeSegment(0, n_local_),
    RAJA_LAMBDA(RAJA::Index_type i)
    {
```
Use spaces instead of tabs for indentation.